app/components/ExtraSettings.vue.ts

```diff
@@ -67,4 +69,10 @@ export default class ExtraSettings extends Vue {
     });
   }
 
+  restartStreamlabelsSession() {
+    if (this.streamlabelsService.restartSession()) {
```

Comment:
This returns a promise, so you will need to wait on the promise to actually make sure it actually succeeded.

Reply:
Done.